src/typedefs.ts

```diff
@@ -55,4 +55,20 @@ export type TransformEvent<T> = TEvent & T & {
   transform: {
     target: any
   }
-}
\ No newline at end of file
+}
+
+export const enum TWebGLPrecision {
+  low = 'lowp',
+  medium = 'mediump',
+  high = 'highp'
+}
+
+/**
+ * @todo remove once rollup supports transforming enums... ANNOYING!
+ * https://github.com/rollup/plugins/issues/463
+ */
```

Comment:
what is the the output so far? isn't TS that supports enum?

Reply:
rollup isn't doing it right and not building an object out of the enum as TS does